.changeset/moody-taxis-study.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/app': minor
```

Comment:
Since this is a bug fix to the localization feature, which is already released, I think we might want to include this in a patch version.